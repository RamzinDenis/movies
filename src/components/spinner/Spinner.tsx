import React from "react";
import { Spin } from "antd";
import { LoadingOutlined } from "@ant-design/icons";

const Spinner = () => {
	const SpinIcon = <LoadingOutlined style={{ fontSize: "40px" }} />;
	return (
		<div>
			<Spin indicator={SpinIcon} />
		</div>
	);
};

export default Spinner;
