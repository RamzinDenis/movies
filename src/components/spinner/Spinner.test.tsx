import React from "react";
import toJson from "enzyme-to-json";
import { shallow } from "enzyme";
import Spinner from "./";

describe("<Spinner />", () => {
	it("should render Spinner correctly", () => {
		expect(toJson(shallow(<Spinner />))).toMatchSnapshot();
	});
});
