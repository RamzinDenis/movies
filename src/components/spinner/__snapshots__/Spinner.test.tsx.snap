// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Spinner /> should render Spinner correctly 1`] = `
<div>
  <Spin
    indicator={
      <ForwardRef(LoadingOutlined)
        style={
          Object {
            "fontSize": "40px",
          }
        }
      />
    }
    size="default"
    spinning={true}
    wrapperClassName=""
  />
</div>
`;
